glade/spectrum_analyzer.glade

```diff
@@ -94,6 +95,7 @@
                     <property name="visible">True</property>
                     <property name="can-focus">True</property>
                     <property name="invisible-char">•</property>
+                    <property name="width-chars">10</property>
```

Comment:
just a nit about this one. You eventually mention that other plugins should care about this but your commit message is misleading in the sense that it looks like you're doing this for all plugins which is not the case. Hence, I would do one of two:

* Either update all plugins in one patch;
* Divide the current patch in 2 for each individual plugin you're addressing.

Reply:
I divided the patch in 2, thanks